src/storage/schema.ts

```diff
@@ -0,0 +1,70 @@
+// import {LabelPreference} from '@atproto/api'
+
+/**
+ * The shape of the object we store in local storage
+ */
+export type Schema = {
+  shell: {
+    colorMode: 'system' | 'light' | 'dark'
+  }
+  // session: {
```

Comment:
The rest of this file is based on `localStorage.getItem('root')` dump in the browser. Actually pretty slim.

Reply:
Left it commented out bc we should double check this before we start using.